tests/objc-cli/test-managed.m

```diff
@@ -22,17 +22,21 @@ int main ()
 		NSLog (@"%s instance property getter", [query answer] == 42 ? "[PASS]" : "[FAIL]");
 		query.answer = 911;
 		NSLog (@"%s instance property setter", [query answer] == 911 ? "[PASS]" : "[FAIL]");
+		[query release];
```

Comment:
You mean we should put this in bindings.h?

```c
#if !__has_feature(objc_arc)
#error Embeddinator code must be built with ARC.
#endif
```

IMHO that's a bit harsh, but I guess we can always remove it later if it becomes a problem.